src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -59,7 +59,7 @@ the main body of the message as well as a quote.
 					{{ messageTime }}
 				</h6>
 				<Actions
-					v-show="showActions && hasActions"
+					v-show="showActions && hasActions && !isConversationReadOnly"
```

Comment:
This should go into `isReplyable` instead. Other (future) actions might still be possible

Reply:
adjusted, moved into the single action as attribute